cloud_formation/scenarios/mid-production.yml

```diff
@@ -2,15 +2,17 @@
 
 # Instance types
 
-ENDPOINT_TYPE: m5.2xlarge
+#ENDPOINT_TYPE: m5.2xlarge
+ENDPOINT_TYPE: m5.xlarge
 RDS_TYPE: db.t3.micro
 REDIS_CACHE_TYPE: cache.m5.xlarge
 REDIS_SESSION_TYPE: cache.t2.small
 REDIS_THROTTLE_TYPE: cache.t2.small
 REDIS_TYPE: cache.t2.small
 CACHE_MANAGER_TYPE: t3.micro
 VAULT_TYPE: m5.large
-ACTIVITIES_TYPE: m5.large
+ACTIVITIES_TYPE: r5.xlarge
```

Comment:
We're ok with the cost for r5.xlarge over an m5.large being 2.6 times more?

Reply:
I think for now we are.  After we get Jordan's update in and downsampled we could go down to r5.large if it makes sense.